File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let len=nums.length;
    let sum;
    for(let i=0;i<len;i++){
        for(let j=i+1;j<len;j++){
            sum=nums[i]+nums[j];
            if(sum==target){
                return [i,j];
            }
        }
    }
};

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map <int,int> um;
        int diff;
        int length=nums.size();
        for(int i=0;i<length;i++){
            diff=target-nums[i];
            if(um.count(diff)){
                return {i,um[diff]};
            }
            um[nums[i]]=i;
        }
        return {};
    }
};